parm/rrfs_mpas_postcntrl.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <postxml>
     <paramset>
-    <datset>WRFTWO</datset>
+    <datset>MPASTWO</datset>
```

Comment:
@Jeff-Duda This file should be saved under UPP/parm/mpas/. A corresponding text file can also be generated via prompting 'make' under UPP/parm.  and both should be committed.